bzt/engine/engine.py

```diff
@@ -95,6 +95,7 @@ def configure(self, user_configs, read_config_files=True):
         :type read_config_files: bool
         """
         self.log.info("Configuring...")
+        self.config['user-configs'] = [os.path.abspath(conf) for conf in user_configs]
```

Comment:
I doubt it's good idea to use engine config as shared memory area.

Reply:
Sure, I have added a new instance variable then